src/Psalm/Internal/Analyzer/Statements/Expression/AssertionFinder.php

```diff
@@ -533,7 +564,7 @@ private static function scrapeInequalityAssertions(
         }
 
         $count = null;
-        $count_inequality_position = self::hasNotCountEqualityCheck($conditional, $count);
+        $count_inequality_position = self::hasCountEqualityCheck($conditional, $count);
```

Comment:
The var (or method) naming here is quite confusing

Reply:
Well, the method in itself doesn't check for identical/equal nor NotIdentical/NotEqual but in scrapeEqualityAssertions we check equality and in scrapeInequalityAssertions we check inequality. Do you have a better suggestion?

Reply:
I think it can stay as is, with the updated docblock.